assets/php/class-wgpb-block-grid-base.php

```diff
@@ -227,6 +228,10 @@ protected function get_container_classes() {
 			$classes[] = "align{$this->attributes['align']}";
 		}
 
+		if ( ! empty( $this->attributes['alignButtons'] ) ) {
+			$classes[] = 'align-cart-buttons';
```

Comment:
Let's keep the `is-/has-*` standard (`align*` comes from wp core), so maybe `has-aligned-buttons`?